ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1481,6 +1492,54 @@ namespace OrdinalAxis {
             return ret;
         }
 
+        /**
+         * If overscroll is pixel or pecentage value, convert it to axis range.
+         *
+         * @private
+         * @param {number | string} overscroll
+         * Overscroll value in axis range, pixels or percentage value.
+         * @return {number}
+         * Overscroll value in axis range.
+         */
+        public convertOverscroll(overscroll : number | string = 0) : number {
+            const ordinal = this,
+                axis = ordinal.axis,
+                overscrollValue = parseInt(overscroll as any, 10);
+
+            if (!defined(axis.max) || !defined(axis.min)) {
+                return 0;
+            }
+
+            if (/%$/.test(overscroll as any)) {
+                // If overscroll is percentage
+                return (axis.max - axis.min) *
+                    overscrollValue / 100;
```

Comment:
I thought the percentage was supposed to work in relation to the length of the axis, not the range 🤔 Could you confirm @TorsteinHonsi ?

Reply:
If it should be 50% of the range, then somehow we have to resolve the problem of mismatching overscroll in navigator and base Axis: https://jsfiddle.net/BlackLabel/t9mpac14/4/ - try panning in the middle chart

Reply:
PS: We have in Utils `relativeLength` method, but I don't see at this moment a huge help from that method

Reply:
> I thought the percentage was supposed to work in relation to the length of the axis, not the range

That was what I was thinking too. But I think you guys should decide what makes more sense. My initial thought was that a percentage setting would make it more predictable in responsive charts, so I guess percentage of the length of the axis would make sense in that case.